lib/tests.js

```diff
@@ -44,7 +44,9 @@ export async function runTests({
   // modules, filtered by the pattern specified by the `--grep` CLI option.
   const testFiles = [
     bootstrapFile,
-    ...glob.sync(testsPattern).filter(path => (grep ? path.match(grep) : true)),
+    ...globSync(testsPattern)
+      .filter(path => (grep ? path.match(grep) : true))
+      .sort(),
```

Comment:
See https://github.com/hypothesis/frontend-build/pull/266/files#r1132211499

This is specially important here, because I found out tests in `client` do not pass if they are not run in order. To be checked.

Reply:
> This is specially important here, because I found out tests in client do not pass if they are not run in order. 

We should definitely fix that. If mocha has an option to randomize the order in which test suites run for example, we could enable that.

Reply:
In this context though, it probably is a good idea to either return files in a deterministic order or explicitly shuffle them to randomize the order. So `.sort()`-ing seems good for this PR.

Reply:
> If mocha has an option to randomize the order in which test suites run for example, we could enable that.

Agree. Having this handled by the test runner would be the ideal solution.